pkg/logging/log.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/fatih/color"
+)
+
+func Info(output string) {
+	color.Blue("Info:")
+	fmt.Printf("%s\n", output)
```

Comment:
With this pattern I believe this will allow you to get rid of all the `Sprintf()` calls when the logging methods are called. 
```suggestion
	fmt.Printf(output, args...)
```

Reply:
All the Info, Warning, Error, and Fatal methods here should follow the same pattern.

Reply:
done